tools/astylerc

```diff
@@ -4,7 +4,7 @@
 style=attach
 
 # Indent with a width of 4 spaces. Use 8 space tabs if possible
-indent=force-tab-x=8
+# indent=force-tab-x=8
```

Comment:
Please do not change astylerc.

Reply:
Those two lines together causes strange formating. 
This is how example function looks like when both of them are enabled
![obraz](https://github.com/Tlf/tlf/assets/1925343/09994971-1a71-4336-ad04-01090c77db79)

and now how when one of them is disabled
![obraz](https://github.com/Tlf/tlf/assets/1925343/999e9dea-d9a8-4a0d-abf5-abf33d0a8fc0)

Sorry but for me looks like something is not right with astyle or that setup.

Reply:
Both options works together only when `indent=force-tab-x` is set to number that is not a multiplayer of 4 (when  at the same time 'indent=force-tab` is set to 4

Reply:
The issue is with the IDE you are using. Vim shows everything as expected:
![image](https://github.com/Tlf/tlf/assets/15141948/b73e4166-75eb-4575-b4a8-0a22a0214a5f)

Try with these VS Code settings:
```
$ cat .config/Code/User/settings.json
{
    "editor.tabSize": 8,
    "editor.insertSpaces": true,
    "editor.indentSize": 4,
    "editor.detectIndentation": false
}
```

![image](https://github.com/Tlf/tlf/assets/15141948/90d5c025-c563-4959-83be-b9fad7acb135)